packages/docs/src/pages/en/components/banners.md

```diff
@@ -14,17 +14,42 @@ related:
 
 The `v-banner` component is used as a middle-interrupting message to the user with one to two actions. It comes in two variations **single-line** and **multi-line** (implicit). These can have icons and actions that you can use with your message. Note that only one banner should be shown at a time.
 
-<entry />
+![Banner Entry](https://cdn.vuetifyjs.com/docs/images/components-temp/v-banner/v-banner-entry.png)
+
+---
 
 ## Usage
 
 Banners can contain one to two lines of text, to which actions and icons can be added.
 
-<!-- <usage name="v-banner" /> -->
+<usage name="v-banner" />
+
+<entry />
+
+## Anatomy
+
+The recommended placement of elements inside of `v-banner` is:
+
+* Place a `v-avatar` or `v-icon` on the far left
+* Place `v-banner-text` to the right of any visual content
+* Place `v-banner-actions` to the far right of textual content, offset bottom
+
+![Banner Anatomy](https://cdn.vuetifyjs.com/docs/images/components-temp/v-banner/v-banner-anatomy.png)
+
+| Element / Area | Description |
+| - | - |
+| 1. Container | The Banner container holds all `v-banner` components |
+| 2. Avatar / Icon (optional) | Leading media content intended to improve visual context |
+| 3. Text | A content area for displaying text and other inline elements |
+| 4. Actions (optional) | A content area that typically contains one or more [v-btn](/components/buttons) components |
 
 ## API
 
-<api-inline />
+| Component | Description |
+| - | - |
+| [v-banner](/api/v-banner/) | Primary Component |
+| **v-banner-text** | Functional component used to display the Banner's subtitle. Wraps the `#text` slot. |
+| **v-banner-actions** | Functional component that modifies the default styling of [v-btn](/components/buttons/) and provides a container for user actions |
```

Comment:
What do you mean when you call these `functional components`?

Reply:
because they are a very specific thing https://vuejs.org/guide/extras/render-function.html#functional-components which those are not

Reply:
![image](https://user-images.githubusercontent.com/9064066/162997835-3fa1f256-5c75-4650-9269-68c957eff0bf.png)

Same thing as v2

Reply:
oh, "functional components" aren't Vue "functional components"??? That's _really_ confusing...

Reply:
Keep in mind this is a port from v2 and in v2 it did have the option **functional: true**.